### DISCUSSION TITLE
Very little little reverb

### DISCUSSION THREAD

**Discussion by TheLemonbird** [djpadbit/Sound-Physics#64] [General]
So i used this fork way back in 2019 when it removed the need for the mono resource pack. and just found that its been updated to 1.16.3. but there is little to no reverb or echo. is there some settings i need to change and why does the settings not work in forge ui? thanks

**Comment by TheLemonbird** [comment]
And thanks for keeping it up to date. this mod is essential and should really be part of the game like pistons and 3d items

**Reply by djpadbit** [reply]
Yea, as i said in https://github.com/djpadbit/Sound-Physics/issues/58, i don't really like how the 1.13+ versions turned out. The openal version changed and the reverb too, so i can't do much about that. It's got the same parameters as in 1.12.2 but it just sounds different. And the Forge config UI is not implemented nicely so there isn't one yet. But you can edit the config with a text editor and if you save it updates in-game.

**Reply by djpadbit** [reply]
I'll even go as far as to say, check out [Dynamic Surroundings](https://www.curseforge.com/minecraft/mc-mods/dynamic-surroundings)'s new 1.16 beta version, it includes sound physics as a module and it's pretty much this minus the auto downmix (but in 1.13+, it's less useful, more for mods) plus all the stuff from Dynamic Surroundings which is an amazing mod that i highly recommend. I always use Sound Physics and Dynamic Surroundings together so it's actually pretty nice. You can still run DS and SP together in 1.16 but you have to disable DS's SP-like thing. And also it's got better mod compatibility, currently SP in 1.16 has a pretty big compatibility issue i still need to fix.

**Reply by TheLemonbird** [reply]
Thanks man i checked it out and its really nice!
